docs/operational_guide/availability_consistency_durability.md

```diff
@@ -0,0 +1,126 @@
+
+## Tuning for Consistency and Durability
+
+### Client Write and Read consistency
+
+The most important thing to understand is that **if you want to guarantee that you will be able to read the result of every successful write, then both writes and reads must be done with `majority` consistency.**
+This means that both writes *and* reads will fail if a quorum of nodes are unavailable for a given shard.
+You can read about the consistency levels in more detail in [the Consistency Levels section](../m3db/architecture/consistencylevels.md)
+
+### Commitlog Configuration
+
+M3DB supports running the commitlog synchronously such that every write is flushed to disk and fsync'd before the client receives a successful acknowledgement, but this is not currently exposed to users in the YAML configuration and generally leads to a massive performance degradation.
+We only recommend operating M3DB this way for workloads where data consistency and durability is strictly required, and even then their may be better alternatives such as running M3DB with the bootstrapping configuration: `filesystem,peers,uninitialized_topology` as described in our [bootstrapping operational guide](./bootstrapping.md).
+
+
+### Writing New Series Asynchronously
+
+If you want to guarantee that M3DB will immediately allow you to read data for writes that have been acknowledged by the client, including the situation where the previous write was for a brand new timeseries, then you  will need to change the default M3DB configuration to set `writeNewSeriesAsync: false` as a top-level key under the `db` section:
+
+```
+writeNewSeriesAsync: false
+```
+
+This instructs M3DB to handle writes for new timeseries (for a given time block) synchronously. Creating a new timeseries in memory is much more expensive than simply appending a new write to an existing series, so this configuration could have an adverse effect on performance when many new timeseries are being inserted into M3DB concurrently.
+
+Since this operation is so expensive, M3DB allows operator to ratelimit the number of new series that can be created per second via the following configuration (also a top-level key under the `db` section):
+
+```
+writeNewSeriesLimitPerSecond: 1048576
+```
+
+### Ignoring Corrupt Commitlogs on Bootstrap
+
+As described in the "Tuning for Performance and Availability" section, we recommend configuring M3DB to ignore corrupt commitlog files on bootstrap. However, if you want to avoid any amount of inconsistency or data loss, no matter how minor, then you should configure M3DB to return unfulfilled when the commitlog bootstrapper encounters corrupt commitlog files. You can do so by modifying your configuration to look like this:
```

Comment:
Hmm do we need both this section and `### Ignoring Corrupt Commitlogs on Bootstrap`?

Reply:
Not totally following. I have the subheading twice, once under availability section and once under consistency